tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -38,6 +43,14 @@
 from lightly.openapi_generated.swagger_client.models.write_csv_url_data import WriteCSVUrlData
 
 
+def _check_dataset_id(dataset_id: str):
+    assert isinstance(dataset_id, str)
+    assert len(dataset_id) > 0
```

Comment:
simply `assert dataset_id` would do here

Reply:
Yes, this is true. However, I like the more explicit version.